pallets/automation-time/src/lib.rs

```diff
@@ -787,52 +798,62 @@ pub mod pallet {
 		/// 
 		/// A task has been completed if all execution times exist in the past and/or are in the current time slot.
 		/// This means there a no more execution times to be run.
-		fn is_completed_task(execution_times: BoundedVec<UnixTime, T::MaxRecurringTimes>) -> bool {
+		fn is_completed_task(execution_times: Vec<UnixTime>) -> bool {
 			let current_time_slot = match Self::get_current_time_slot() {
 				Ok(time_slot) => time_slot,
-				Err(_) => return false, // fix error
+				// This will only occur for the first block in the chain.
+				Err(_) => return false,
 			};
 
 			let scheduled_times = Self::clean_execution_times_vector(execution_times);
-			return *scheduled_times.last().unwrap() <= current_time_slot;
+			let is_complete = *scheduled_times.last().unwrap() <= current_time_slot;
+			return is_complete;
 		}
 
 		/// Removes the task of the provided task_id and all scheduled tasks, including those in the task queue.
 		///
-		fn remove_task(task_id: T::Hash, task: Task<T>) {
-			let current_time_slot = match Self::get_current_time_slot() {
-				Ok(time_slot) => time_slot,
-				Err(_) => return, // fix error
-			};
+		fn remove_task(task_id: T::Hash, task: Task<T>) -> Result<(), Error<T>> {
```

Comment:
oh huh, where did you add the new error? thought the errors in this function are bubbled up as events

Reply:
old commit?